config/containers/logging/dual-logging.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Learn how to read container logs locally when using a third party logging solution.
+keywords: docker, logging, driver
+title: Reading container logs locally with remote logging drivers
+---
+
+## Overview 
+
+In previous versions of Docker Enterprise, the jsonfile and journald log drivers supported reading 
```

Comment:
"Docker Enterprise" -> "Docker Engine Enterprise"

Reply:
Got it.